sdk/core/azure-core-v2/src/main/java/com/azure/v2/core/credentials/TokenRequestContext.java

```diff
@@ -22,17 +22,17 @@
  * </p>
  *
  * <p>
- * Here's a high-level overview of how you can use the {@link AzureTokenRequestContext}:
+ * Here's a high-level overview of how you can use the {@link TokenRequestContext}:
  * </p>
  *
  * <ol>
- * <li>Create an instance of the {@link AzureTokenRequestContext} class and configure the required properties.
- * The {@link AzureTokenRequestContext} class allows you to specify the scopes or resources for which you want to request
+ * <li>Create an instance of the {@link TokenRequestContext} class and configure the required properties.
+ * The {@link TokenRequestContext} class allows you to specify the scopes or resources for which you want to request
  * an access token, as well as any additional claims or options.</li>
  *
  * <li>Pass the TokenRequestContext instance to the appropriate authentication client or mechanism when
  * requesting an access token. The specific method or API to do this will depend on the authentication mechanism
- * you are using. For example, if you are using Azure Identity for MS Entra authentication, you would pass the
+ * you are using. For example, if you are using Azure Identity for Microsoft Entra authentication, you would pass the
  * AzureTokenRequestContext instance to the getToken method of the {@link TokenCredential} implementation.</li>
```

Comment:
Same comment as above - I know this is largely unchanged from what we have today (we should clean that up too)